run_buyer_frontend.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
:+1: on having a simple `run_something.sh` script that adds sensible defaults. I wonder if we should call it the same in every app (something like `run_app.sh`). It would mean that we could more easily write tools to automate startup for development.